website_sale_secondary_unit/views/templates.xml

```diff
@@ -0,0 +1,210 @@
+                                t-att-data-secondary-uom-factor="secondary_uom.factor"
+                                t-att-data-product-uom-factor="product.sudo().uom_id.factor"
+                            >
+                                <t
+                                    t-call="website_sale_secondary_unit.second_qty_description"
+                                />
+                            </option>
+                        </t>
+                    </select>
+                </div>
+            </t>
+        </xpath>
+    </template>
+    <template id="product_price" inherit_id="website_sale.product_price">
+        <xpath expr="//span[hasclass('oe_price')]" position="after">
+            <t t-if="secondary_uom_ids">
+                / <span
+                    class="css_editable_mode_hidden price_uom"
+                    t-field="product.uom_id"
+                />
+            </t>
+        </xpath>
+    </template>
+    <template id="product_quantity" inherit_id="website_sale.product_quantity">
+        <xpath expr="//input[@name='add_qty']/.." position="attributes">
+            <attribute
+                name="t-attf-class"
+            >css_quantity input-group {{'d-none' if combination_info['prevent_zero_price_sale'] or secondary_uom_ids else 'd-inline-flex'}} me-2 mb-2 align-middle</attribute>
+        </xpath>
+    </template>
+    <template id="cart_lines" inherit_id="website_sale.cart_lines">
+        <xpath expr="//div[@name='website_sale_cart_line_quantity']" position="before">
+            <t t-if="line.secondary_uom_id">
+                <div class="css_secondary_quantity css_quantity input-group mb-2">
+                    <a
+                        t-attf-href="#"
+                        class="js_add_cart_json btn btn-link d-inline-block border-end-0"
+                        aria-label="Remove one"
+                        title="Remove one"
+                    >
+                        <i class="position-relative z-index-1 fa fa-minus" />
+                    </a>
+                    <input
+                        type="text"
+                        class="js_secondary_quantity secondary-quantity form-control border-start-0 border-end-0"
+                        t-att-data-line-id="line.id"
+                        t-att-data-product-id="line.product_id.id"
+                        t-att-data-secondary-uom-id="line.secondary_uom_id.id"
+                        t-att-data-secondary-uom-factor="line.secondary_uom_id.factor"
+                        t-att-data-product-uom-factor="line.product_uom.sudo().factor"
+                        t-att-value="int(line.secondary_uom_qty) == line.secondary_uom_qty and int(line.secondary_uom_qty) or line.secondary_uom_qty"
+                    />
+                    <a
+                        t-attf-href="#"
+                        class="js_add_cart_json d-inline-block float_left btn btn-link border-start-0"
```

Comment:
```suggestion
                        class="js_add_cart_json d-inline-block float_start btn btn-link border-start-0"
```
It would be necessary to check if the design is affected since the class is not being applied, perhaps it is not necessary in this version.